rules.md

```diff
@@ -0,0 +1,18 @@
+# Substrate
+
+- 🔒️ed lines are reviewed by 2 of {@gavofyork, @shawntabrizi, @rphmeier, @bkchr}
+- PRs are reviewed by 2 of @paritytech/core-devs
```

Comment:
This does not include pinging.

Reply:
as per our discussion, this can be done by **not assigning** the team, but _passively-aggressively waiting and failing a job until someone suitable reviews a pr_. Details may be shown in the job's output.